gpg/gpg_test.go

```diff
@@ -166,13 +167,18 @@ func TestGPGCheckMessageSignature(t *testing.T) {
 
 func TestGPGEncryptMessage(t *testing.T) {
 	gpg := setupGPG(t)
+	assert.NotNil(t, gpg)
```

Comment:
This check is not really necessary, since that case would(should) have been caught in setupGPG.